cookbook/security/access_control.rst

```diff
@@ -54,12 +54,10 @@ Take the following ``access_control`` entries as an example:
 
             <config>
                 <!-- ... -->
-                <access-control>
-                    <rule path="^/admin" role="ROLE_USER_IP" ip="127.0.0.1" />
-                    <rule path="^/admin" role="ROLE_USER_HOST" host="symfony\.com$" />
-                    <rule path="^/admin" role="ROLE_USER_METHOD" method="POST, PUT" />
-                    <rule path="^/admin" role="ROLE_USER" />
-                </access-control>
+                <rule path="^/admin" roles="ROLE_USER_IP" ip="127.0.0.1" />
+                <rule path="^/admin" roles="ROLE_USER_HOST" host="symfony\.com$" />
+                <rule path="^/admin" roles="ROLE_USER_METHOD" methods="POST, PUT" />
+                <rule path="^/admin" roles="ROLE_USER" />
```

Comment:
Actually, I am not sure if we should really use the `rule` element at all. Would it be more clear if we simply used multiple `access-control` elements?